servant-util/src/Servant/Util/Combinators/Pagination.hs

```diff
@@ -13,10 +13,12 @@ import Control.Lens ((<>~), (?~))
 import Data.Default (Default (..))
 import qualified Data.Swagger as S
 import qualified Data.Text as T
-import Servant ((:>), HasServer (..), QueryParam)
+import Servant ((:>), DefaultErrorFormatters, ErrorFormatters, HasContextEntry, HasServer (..),
+                QueryParam)
```

Comment:
Btw, please enable `stylish-haskell` on save and reformat all the imports in the modules you update in this PR. I get extra diff when trying to update this module somehow.

Reply:
AFAIK Sasha uses version 0.11, that may be the reason.

Reply:
Oh really, that's a nice meme indeed :facepalm: 

I use 0.12.2.0.

I thought only ormolu changes its behaviour depending on version, not on config, so :fuuu: if stylish-haskell also suffers from this.

Reply:
In 0.12 they [switched](https://github.com/jaspervdj/stylish-haskell/blob/main/CHANGELOG) to `ghc-lib-parser` from `haskell-src-exts`, so I think such minor inconsistencies are somewhat expected and justified. That sounds like a very crucial change, but a good one.